src/wx/guiinit.cpp

```diff
@@ -2244,11 +2244,12 @@ void MainFrame::add_menu_accels(wxTreeCtrl* tc, wxTreeItemId& parent, wxMenu* me
             // but then if the user overides main menu, that req. is broken..
             wxString txt = (*mi)->GetItemLabelText();
 
-            for (int i = 0; i < 10; i++)
-                if (*mi == loadst_mi[i] || *mi == savest_mi[i]) {
-                    txt = cmdtab[i].name;
```

Comment:
Here. We are using the wrong index. `cmdtab[i]` is not where the save/load state is. We needed to get the label (`LoadGame01`, for example), then go into the table and check which command has this name and then get the label.